app/models/concerns/client_search.rb

```diff
@@ -89,19 +86,5 @@ def self.metaphone_search(where, field, text)
 
       where
     end
```

Comment:
Was metaphone search completely removed? I think thats probably fine just checking.

Reply:
Metaphone search should still work although maybe we should just replace it completely here. The current metaphone search isn't great for realtime data since the names are only built once a day.

Word similarity may be better in some cases (typos or non-western names) but it might be worse when the phonetic is different form the spelling (phu vs foo). With this construction we could support both metaphone and word similarity in the name search. We probably would add generated metaphone columns on the db rather than building uniqueNames in the app